assets/sass/05-blocks/utilities/_editor.scss

```diff
@@ -15,7 +15,7 @@
 		font-family: var(--heading--font-family);
 		font-weight: var(--heading--font-weight-page-title);
 		font-size: var(--global--font-size-page-title);
-		line-height: var(--heading--line-height);
+		line-height: var(--heading--font-size-h1);
```

Comment:
I think it is because the line height is set to a font size, not the line height.